go.mod

```diff
@@ -16,6 +16,8 @@ replace (
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.3
 	github.com/googleapis/gax-go/v2 => github.com/googleapis/gax-go/v2 v2.1.1
 	github.com/juju/errors => github.com/k3s-io/nocode v0.0.0-20200630202308-cb097102c09f
+
+	github.com/k3s-io/helm-controller => github.com/acorn-io/helm-controller v0.0.0-20230626182853-bb03cf954008
```

Comment:
Why are we now replacing a k3s-io org repo with an external one?

Reply:
I believe this is just to test the change before  https://github.com/k3s-io/helm-controller/pull/198 is merged.

Reply:
This will be changed once the dependency PR is merged

Reply:
https://github.com/k3s-io/helm-controller/releases/tag/v0.15.2